src/projectFormController.js

```diff
@@ -58,42 +58,45 @@ function createSelect(target, id, options) {
   });
 }
 
-function createProjectInputs() {
+function createInputFields(fields, target) {
+  fields.forEach(field => {
+    let type = 'text';
+    let fieldName;
+
+    if (field.includes('date')){ 
+      type = 'date'; 
+    }
+    
+    if (field.includes('-')) {
+      fieldName = field.split('-').join(' ');
```

Comment:
Clever girl! I like how you did this with the split, although you can actually remove the `if` part because if there's no `-` then it just won't split anything and will return `field` again. This can then be changed so that line 64 is `const fieldName = field.split('-').join(' ');`.

As an aside learning point, if we pretend the other isn't true, then I would have done it with a shorter ternary here instead of if/else:
```js
const fieldName = field.includes('-') ? field.split('-').join(' ') : field;
```